fitbenchmarking/parsing/fitbenchmark_parser.py

```diff
@@ -1,10 +1,11 @@
 """
 This file implements a parser for the Fitbenchmark data format.
 """
-from collections import OrderedDict
-
 import os
 import typing
+from collections import OrderedDict
```

Comment:
I thihnk this is still used elsewhere in the file, it only shows up as a change because the autoformatter moved it I think. This should probably be included in a new issue like #1122